docs/src/accessibility-testing-java.md

```diff
@@ -0,0 +1,238 @@
+
+You can find the rule IDs to pass to `disableRules()` in the `id` property of the violations you want to suppress. A [complete list of axe's rules](https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md) can be found in `axe-core`'s documentation.
+
+```java
+AxeResults accessibilityScanResults = new AxeBuilder(page)
+  .disableRules(Arrays.asList("duplicate-id"))
+  .analyze();
+
+assertEquals(Collections.emptyList(), accessibilityScanResults.getViolations());
+```
+
+### Using violation fingerprints to specific known issues
+
+If you would like to allow for a more granular set of known issues, you can use the following pattern:
+
+1. Perform an accessibility scan which is expected to find some known violations
+1. Convert the violations into "violation fingerprint" objects
+1. Assert that the set of fingerprints is equivalent to the expected ones
+
+This approach avoids the downsides of using `AxeBuilder.exclude()` at the cost of slightly more complexity and fragility.
+
+Here is an example of using fingerprints based on only rule IDs and "target" selectors pointing to each violation:
+
+```java
+@Test
+shouldOnlyHaveAccessibilityViolationsMatchingKnownFingerprints() throws Exception {
+  page.navigate("https://your-site.com/");
+
+  AxeResults accessibilityScanResults = new AxeBuilder(page).analyze();
+
+  List<ViolationFingerprint> violationFingerprints = fingerprintsFromScanResults(accessibilityScanResults);
+          
+  assertEquals(Arrays.asList(
+    new ViolationFingerprint("aria-roles", "[span[role=\"invalid\"]]"),
+    new ViolationFingerprint("color-contrast", "[li:nth-child(2) > span]"),
+    new ViolationFingerprint("label", "[input]")
+  ), violationFingerprints);
+}
+
+// You can make your "fingerprint" as specific as you like. This one considers a violation to be
+// "the same" if it corresponds the same Axe rule on the same element.
+//
+// Using a record type makes it easy to compare fingerprints with assertEquals
+public record ViolationFingerprint(String ruleId, String target) { }
+
+public List<ViolationFingerprint> fingerprintsFromScanResults(AxeResults results) {
+  return results.getViolations().stream()
+    // Each violation refers to one rule and multiple "nodes" which violate it
+    .flatMap(violation -> violation.getNodes().stream()
+      .map(node -> new ViolationFingerprint(
+        violation.getId(),
+        // Each node contains a "target", which is a CSS selector that uniquely identifies it
+        // If the page involves iframes or shadow DOMs, it may be a chain of CSS selectors
+        node.getTarget().toString()
+      )))
+    .collect(Collectors.toList());
+}
+```
+
+## Using a test fixture for common axe configuration
+
+A [`TestFixtures` class](./test-runners#running-tests-in-parallel) is a good way to share common `AxeBuilder` configuration across many tests. Some scenarios where this might be useful include:
```

Comment:
Playwright Java currently doesn't provide standard fixtures (with page, context, browser etc objects) and the reference to the test runner section may be confusing. We may add such fixtures in the future (discussed this just today in the team meeting) but before that perhaps it's better to just recommend moving the code into a shared utility method and have the callsite pass `page` as an explicit argument `makeAxeBuilder(page)` ?